Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -51,8 +51,8 @@
 
 upgradeProperties[2017] = {
     '2017' : {
-        'Geom' : 'DB:Extended',
-        'GT' : 'auto:phase1_2017_realistic',
+        'Geom' : 'Extended2017dev',
+        'GT' : 'auto:phase1_2017_hcaldev',
```

Comment:
> There are aso BPIX changes that need validation in the normal 2017 geometry.

It is not just the BPix changes we need to validate with pre12, but also all non-geometry changes between pre11 and pre12 (e.g. turning off VPF30 mitigation), and that is best done with exactly the same geometry as in pre11.

Now, trying to be practical (probably along what @slava77 already wrote), as these two parameters are passed to cmsDriver they can be changed at the time of RelVal submission (to be coordinated with PdmV; by the way, if this PR goes in pre12, do you want RelVals with the `Extended2017dev` geometry?).

So my only concern is the effect of the `run2_HF_2017` and `run2_HCAL_2017` sub-eras when run on 2016 HCAL geometry (both when re-using pre11 GEN-SIM and generating new). If the effect is statistically insignificant (within the usual 9k events) on high-pT jets (i.e. jetCore tracking iteration and pixel cluster splitting), then I'm personally fine with changing the "2017" workflow geometry to `Extended2017dev` for pre12.